tools/cloud-build/daily-tests/blueprints/lustre-with-new-vpc.yaml

```diff
@@ -49,8 +49,8 @@ deployment_groups:
     source: modules/scripts/startup-script
     settings:
       runners:
-      - $(scratchfs.install_ddn_lustre_client_runner)
-      - $(scratchfs.mount_runner)
+      - $(scratchfs.network_storage.client_install_runner)
```

Comment:
You can get rid of the use of startup script completely here and just use the file system downstream. They were kept here to test the runners that you are deleting. See #1429 for example.